### PR TITLE
[IMP] stock: clarify procurement rule message

### PR DESCRIPTION
**Pull request by mgh-odoo**
**Task**: https://www.odoo.com/web#id=1960746&model=project.task&menu_id=4720
**Pad**: https://pad.odoo.com/p/r.79c217495d11c87d4704965f3b389c6a

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock/models/stock_rule.py [hunk 1]
```diff
@@ -348,8 +348,8 @@ def run(self, procurements):
             procurement.values.setdefault('date_planned', fields.Datetime.now())
             rule = self._get_rule(procurement.product_id, procurement.location_id, procurement.values)
             if not rule:
-                errors.append(_('No procurement rule found in location "%s" for product "%s".\n Check routes configuration.') %
-                    (procurement.location_id.display_name, procurement.product_id.display_name))
+                errors.append(_('No rule has been found to pull "%s" ProductName in "%s".\nVerify the routes configuration on the product.') %
```
Comment (sswapnesh) [inline addons/stock/models/stock_rule.py]:
```suggestion
                errors.append(_('No rule has been found to pull "%s" in "%s".\nVerify the routes configuration on the product.') %
```

**Comment by amoyaux** [comment]
robodoo r+

**Comment by amoyaux** [comment]
robodoo retry

**Comment by sswapnesh** [comment]
This one need @robodoo r+

**Comment by amoyaux** [comment]
robodoo retry

**Comment by amoyaux** [comment]
robodoo r+

**Comment by amoyaux** [comment]
@mgh-odoo Thanks for rebase

**Comment by amoyaux** [comment]
robodoo retry

**Comment by amoyaux** [comment]
robodoo r+

**Comment by amoyaux** [comment]
robodoo retry

**Comment by rdeodoo** [comment]
@amoyaux This one stuck also, but task is set as merged.
Maybe wait code is merged before set task deployed

**Comment by amoyaux** [comment]
robodoo r+
@rdeodoo Ouep, merci pour le rappel.

**Comment by rdeodoo** [comment]
`rte_translator` :nauseated_face:

**Comment by sswapnesh** [comment]
Keep calm and @robodoo r+ again

**Comment by amoyaux** [comment]
robodoo retry

**Pull request closed without merging**